test/ng/directive/formSpec.js

```diff
@@ -290,7 +290,7 @@ describe('form', function() {
   describe('triggering commit value on submit', function() {
     it('should trigger update on form submit', function() {
       var form = $compile(
-          '<form name="test" ng-model-options="{ updateOn: \'\' }" >' +
+          '<form name="test" ng-model-options="{ updateOn: \'submit\' }" >' +
```

Comment:
Good question. It seems that my commits caused a change to the semantic of providing an empty string to the `updateOn` property. We never documented what an empty string should do, and I honestly don't know why these tests used an empty string. I think they are trying to "never" update the model, right?